packages/request-client.js/src/api/request.ts

```diff
@@ -94,11 +93,7 @@ export default class Request {
       requestId: this.requestId,
     };
 
-    try {
-      await this.requestLogic.acceptRequest(parameters, signerIdentity, true);
-    } catch (error) {
-      throw new Error(`${error.message} (maybe transactions are still pending)`);
-    }
+    await this.requestLogic.acceptRequest(parameters, signerIdentity, true);
```

Comment:
why did we remove the error messages?

Reply:
Because now we take in account the pending state